### DISCUSSION TITLE
requiring user/password for server/directory access ?

### DISCUSSION THREAD

**Discussion by strk** [jamulussoftware/jamulus#2990] [Technical Support & Help] [answered]
I'm thinking to setup a directory to more easily make a set of servers discoverable by friends but I want to protect access to the directory and the server to keep it private, is it possible to do any of the two things ?

**Comment by mcfnord** [comment]
The Jamulus software doesn't provide any of these features (password-protected server and password-protected directory). But I bet people have done some of this with other tools, so I hope someone can give some clues.

**Comment by pljones** [comment]
A Directory can be put behind a firewall and access to through the firewall only granted to registered addresses.  Similarly, Servers can be set up in a similar manner.  WorldJam uses this kind of set up, where contributors to each show must "register" for that show to enable access.

**Comment by mcfnord** [comment, marked as answer]
Oh hey, maybe the `--listfilter` option [lets you specify which servers can register with your custom directory](https://jamulus.io/wiki/Command-Line-Options).
